aws/resource_aws_redshift_snapshot_schedule_test.go

```diff
@@ -0,0 +1,455 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/redshift"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_redshift_snapshot_schedule", &resource.Sweeper{
+		Name: "aws_redshift_snapshot_schedule",
+		F:    testSweepRedshiftSnapshotSchedules,
+	})
+}
+
+func testSweepRedshiftSnapshotSchedules(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).redshiftconn
+
+	req := &redshift.DescribeSnapshotSchedulesInput{}
+
+	resp, err := conn.DescribeSnapshotSchedules(req)
+	if err != nil {
+		if testSweepSkipSweepError(err) {
+			log.Printf("[WARN] Skipping Redshift Regional Snapshot Schedules sweep for %s: %s", region, err)
+			return nil
+		}
+		return fmt.Errorf("Error describing Redshift Regional Snapshot Schedules: %s", err)
+	}
+
+	if len(resp.SnapshotSchedules) == 0 {
+		log.Print("[DEBUG] No AWS Redshift Regional Snapshot Schedules to sweep")
+		return nil
+	}
+
+	for _, snapshotSchedules := range resp.SnapshotSchedules {
+		identifier := aws.StringValue(snapshotSchedules.ScheduleIdentifier)
+
+		hasPrefix := false
+		prefixes := []string{"tf-snapshot-schedule-"}
```

Comment:
We have been preferring to omit prefix checking in sweepers since its hard to ensure all contributions to the testing use them 👍